crates/node/src/lib.rs

```diff
@@ -596,7 +596,7 @@ impl ClusterConfiguration {
     pub fn from_configuration(configuration: &Configuration) -> Self {
         ClusterConfiguration {
             num_partitions: configuration.common.default_num_partitions,
-            partition_replication: configuration.admin.default_partition_replication.clone(),
```

Comment:
I don't expect many people to configure this value, but I am wondering whether we shouldn't deprecate it first in 1.3 before removing it in 1.4.

Reply:
My thinking was that this value is only used on first bootstrap, hence "deprecating" it can just that we print the deprecation warning. Anyway, i changed it so we still respect the value if its set